.github/workflows/scan.yml

```diff
@@ -10,6 +10,10 @@ on:
         description: 'Number of supported releases'
         type: number
         default: 1
+    secrets:
+      TOKEN:
+        description: 'Github token to use, optional'
+        required: false
```

Comment:
Can we confirm that this won't print a plain text version of the token in any logs of the CI run if triggered with this input.

Reply:
The token we put in secret is  securely masked by GitHub Actions by default.

![Screenshot 2024-09-30 at 19 54 06](https://github.com/user-attachments/assets/d9f4017a-0063-4e4a-836f-c5cddaf4f18f)